### PR TITLE
Add a description why bluetooth is required

### PR DESCRIPTION
**Pull request by m-kuhn**

### CODE REVIEW

**Review by suricactus — APPROVED** [review]

File: ios/Info.plist.in [hunk 1]
```diff
@@ -73,6 +73,12 @@
     <!-- Copyright -->
     <key>NSHumanReadableCopyright</key>
     <string>${MACOSX_BUNDLE_COPYRIGHT}</string>
+
```
Comment (suricactus) [inline ios/Info.plist.in:76]:
```suggestion

    <!-- Bluethooth perms description -->
```

Comment (suricactus) [reply ios/Info.plist.in:76]: Now it is the "copyright" comments section.

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for a7644806a52def44f2de8c6d24c021d32c4643e1:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1980-7ab1c528814f695b3c338982a7ff8ab0c0790de6-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1980-7ab1c528814f695b3c338982a7ff8ab0c0790de6-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1980-7ab1c528814f695b3c338982a7ff8ab0c0790de6-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1980-7ab1c528814f695b3c338982a7ff8ab0c0790de6-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 8c67dc50ef54d998e54172c240558be53f3caeb1:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1980-33d0b4316e0a2eea85f43fc6c3767c4ee73f9b64-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1980-33d0b4316e0a2eea85f43fc6c3767c4ee73f9b64-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1980-33d0b4316e0a2eea85f43fc6c3767c4ee73f9b64-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1980-33d0b4316e0a2eea85f43fc6c3767c4ee73f9b64-x86.apk)

**Pull request merged**